pages/hobbies.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Instructor Hobbies</title>
+  <link rel="stylesheet" href="../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/hobbies.css">
+</head>
+<body>
+  <header>
+    <div id="header">
+      <h1 >Hobbies: Beyond the Keyboard</h1> 
+    </div>
+  </header>
+  <section class="hobbies-links">
+    <a href="../index.html">Home</a>
+    <a href="education.html">Education</a>
+    <a href="fun_facts.html">Fun facts</a>
+</section>
+      
+<section class="hobbies-container">
+    <article>
+      <h3>Adrian Prado</h3>
+      <div class="instructor-card">
+        <img src="../images/Adrian-Prado-600x600.png" alt="Hey, Adrian!" width="300" height="200">
+        
+        <div class="description">
+          Adrian's hobby is improv. He is a gifted improviser 
+          whose quick wit and creative energy light up the stage. 
+          His ability to craft engaging and humorous scenes 
+          on the spot showcases his sharp mind and charismatic 
+          presence. Beyond improv, Adrian enjoys cooking and baking; 
+          during the pandemic, he started a live baking show on Facebook, 
+          with tres leches being his favorite to bake. 
+          He is also a huge fan of escape rooms, having attempted 
+          almost every escape room in Austin and successfully 
+          beating all but four.
+      </div>
```

Comment:
Since `div` tags aren't semantic, I would suggest changing these to `p` tags instead.
```suggestion
        <p class="description">
          Adrian's hobby is improv. He is a gifted improviser 
          whose quick wit and creative energy light up the stage. 
          His ability to craft engaging and humorous scenes 
          on the spot showcases his sharp mind and charismatic 
          presence. Beyond improv, Adrian enjoys cooking and baking; 
          during the pandemic, he started a live baking show on Facebook, 
          with tres leches being his favorite to bake. 
          He is also a huge fan of escape rooms, having attempted 
          almost every escape room in Austin and successfully 
          beating all but four.
      </p>
```